.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,42 @@
+name: Bug Report🐛
+description: Create a report to help us improve
+title: "Bug🐛: "
+labels: [bug]
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to file a Bug report.
+  - type: textarea
+    attributes:
+      label: Describe the bug.
+      description: "A clear and concise description of what the bug is."
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Describe the steps to reproduce the behavior.
+      description: "Be clear and concise while describing the steps to ensure that it can be reproduced and potentially fixed"
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+  - type: textarea
+    attributes:
+      label: Expected behavior.
+      description: "A clear and concise description of what you expected to happen."
+  - type: textarea
+    attributes:
+      label: Screenshots.
+      description: "If applicable, add screenshots to help explain your problem."
```

Comment:
Add: "You may need to attach any screenshots to the issue after it has been created (after you submit this bug report).